src/modules/previewpane/SvgPreviewHandler/SvgPreviewControl.cs

```diff
@@ -220,20 +233,11 @@ private void AddWebViewControl(string svgData)
                     _browser.CoreWebView2.AddWebResourceRequestedFilter("*", CoreWebView2WebResourceContext.All);
                     _browser.CoreWebView2.WebResourceRequested += CoreWebView2_BlockExternalResources;
 
-                    // WebView2.NavigateToString() limitation
-                    // See https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2.navigatetostring?view=webview2-dotnet-1.0.864.35#remarks
-                    // While testing the limit, it turned out it is ~1.5MB, so to be on a safe side we go for 1.5m bytes
-                    if (svgData.Length > 1_500_000)
```

Comment:
Not sure if it's good to remove the limit? 🤔 What side effects can this change lead to?

Reply:
let's keep it in memory as it was, I saw some workarounds for this webview 2mb limitations, so we might want to try to avoid generating files completely some time soon